src/yuzu/game_list.cpp

```diff
@@ -728,6 +771,57 @@ void GameList::RefreshGameDirectory() {
     }
 }
 
+void GameList::ToggleFavorite(u64 program_id) {
+    if (!UISettings::values.favorited_ids.contains(program_id)) {
+        tree_view->setRowHidden(0, item_model->invisibleRootItem()->index(),
+                                !search_field->SearchFieldEmpty());
+        UISettings::values.favorited_ids.append(program_id);
+        AddFavorite(program_id);
+        item_model->sort(tree_view->header()->sortIndicatorSection(),
+                         tree_view->header()->sortIndicatorOrder());
+    } else {
+        UISettings::values.favorited_ids.removeOne(program_id);
+        RemoveFavorite(program_id);
+        if (UISettings::values.favorited_ids.size() == 0) {
+            tree_view->setRowHidden(0, item_model->invisibleRootItem()->index(), true);
+        }
+    }
+}
+
+void GameList::AddFavorite(u64 program_id) {
+    const auto favorites_row = item_model->item(0);
```

Comment:
ditto with any other instances below
```suggestion
    const auto* favorites_row = item_model->item(0);
```

Reply:
I also removed `const` because `appendRow()` is used below.